### PR TITLE
So nice, we aliased it twice!

### PR DESCRIPTION
**Pull request by theengineear**
This duplication got aliased twice this morning. Just removing the dups.

### CODE REVIEW

File: applications/jupyter-extension/nteract_on_jupyter/webpack.config.js [hunk 1]
```diff
@@ -8,21 +8,23 @@ const isProd = nodeEnv === "production";
...
-  "@nteract/mathjax": "@nteract/mathjax/src",
-  "@nteract/core": "@nteract/core/src",
-  "@nteract/messaging": "@nteract/messaging/src",
-  "@nteract/editor": "@nteract/editor/src",
-  "@nteract/commutable": "@nteract/commutable/src",
-  "@nteract/dropdown-menu": "@nteract/dropdown-menu/src",
-  "@nteract/transform-model-debug": "@nteract/transform-model-debug/src",
-  "rx-jupyter": "rx-jupyter/src",
-  "rx-binder": "rx-binder/src",
-  ...rxAliases
```
Comment (theengineear) [inline applications/jupyter-extension/nteract_on_jupyter/webpack.config.js]: @rgbkrk this line was failing for me when I ran `npm run app:jupyter-extension`. Do we (1) have something transpiling this file and (2) if we do, any thoughts on why it wouldn't be working on my machine?

**Comment by theengineear** [comment]
AH! I'm on Node 8.2.1, I think you might just be ahead of me. Looks like it's supported in Node 8.6.0+. Let me try that real quick.

**Comment by theengineear** [comment]
^^ OK, updated my node and away we go. Duplicate declaration still stands 😛

**Comment by rgbkrk** [comment]
Whoops! Yeah I upgraded recently.

Luckily I've got an automated setup for this that I'll post as a PR soon.

**Pull request merged**